infrastructure/prototype-app/template.yaml

```diff
@@ -163,7 +163,7 @@ Resources:
                 token.actions.githubusercontent.com:aud: sts.amazonaws.com
                 token.actions.githubusercontent.com:sub: !Sub repo:${GitHubOrg}/${PocRepositoryName}:*
       ManagedPolicyArns:
-        - !Ref GitHubActionsPolicy
+        - arn:aws:iam::aws:policy/AdministratorAccess
```

Comment:
Yeah this makes sense and unblocks us on something reasonably small for the prototype. Nice one!

So down the road I guess we'll want to define a `/AutomatedAccess` or something?
Would like to have a 🤔 about what "good" looks like before we do our first prod app.

I guess much of the DI TechOps work will help us there.

Reply:
Yes definitely. I had a bit of a snoop around in the Auth team's setup to see if I could find where their permissions are defined but it's slightly different as they have Github actions -> S3, then a different thing (Code build) takes over to do the deploying. Here's the Github actions permissions for example: https://github.com/alphagov/di-infrastructure/blob/main/terraform/tools/iam.tf

Ours will look more like that once we're using the shared deploy pipeline which will give minimal permissions to Github.